File: java/java-kafka-producer/src/main/java/com/showmeyourcode/kafka/java/kafka_producer/MainProducer.java
package com.showmeyourcode.kafka.java.kafka_producer;

public class MainProducer {
    public static void main(String... args) {
        int numberOfMessages;
        if (args.length == 0) {
            numberOfMessages = 5;
        } else {
            numberOfMessages = Integer.parseInt(args[0]);
        }
        JavaKafkaProducer.runProducer(numberOfMessages);
        JavaKafkaProducerAvroClass.runProducer(numberOfMessages);
        JavaKafkaProducerAvroFile.runProducer(numberOfMessages);
    }
}

File: kotlin/kotlin-kafka-producer/src/main/kotlin/com/showmeyourcode/kafka/kotlin/kafka_producer/main-producer.kt
package com.showmeyourcode.kafka.kotlin.kafka_producer

fun main(args: Array<String>) {
    if (args.isEmpty()) {
        val numberOfMessages = 5
        KotlinKafkaProducer.runProducer(numberOfMessages)
        KotlinKafkaProducerAvro.runProducer(numberOfMessages)
    } else {
        val numberOfMessages = Integer.parseInt(args[0])
        KotlinKafkaProducer.runProducer(numberOfMessages)
        KotlinKafkaProducerAvro.runProducer(numberOfMessages)
    }
}
